conf/muchos.props.example

```diff
@@ -278,10 +278,11 @@ yarn_nm_mem_mb=16384
 # Where:
 #   Hostname = Must be unique.  Will be used for hostname in EC2 or should match hostname on your own cluster
 #   Service = Service to run on node (possible values: zookeeper, namenode, resourcemanager, accumulomaster, client, swarmmanager,
-#             mesosmaster, worker, fluo, metrics, spark). The following services are required: namenode, resourcemanager,
+#             mesosmaster, worker, fluo, metrics, spark, elkserver). The following services are required: namenode, resourcemanager,
 #             accumulomaster, zookeeper & worker
 # If cluster_type=azure, the list of nodes below is auto-generated by the launch action e.g. "muchos launch --cluster accumuloclstr"
 # For the 'azure' cluster type, it is perfectly normal if the auto-generated list of node names is not sequential
+# For viewing logging and statistics with Kibana, the elkserver service must be added to a leader node.
```

Comment:
I think this mention of elkserver depends on PR #338, doesn't it?

Reply:
Yes, I will remove it from here and update #338 with that line,